prime-dotnet-webapi/Models/Documents/SelfDeclarationDocument.cs

```diff
@@ -15,5 +15,6 @@ public class SelfDeclarationDocument : BaseDocumentUpload, IEnrolleeNavigationPr
 
         [JsonIgnore]
         public SelfDeclarationType SelfDeclarationType { get; set; }
+        public bool IsHidden { get; set; }
```

Comment:
"Hidden" already sounds like a boolean, don't really need to call it `IsHidden`.
Also, please observe the spacing of the file.